docs/manual/docs/cmd/graph/reports/reports-teams-device-usage-user-detail.md

```diff
@@ -0,0 +1,48 @@
+# graph reports Microsoft Teams device usage by user
```

Comment:
The .md file name should match command name (so `report-teamsdeviceusageuserdetail`)

Reply:
Fixed and ready for review.